core/chains/evm/logpoller/log_poller_test.go

```diff
@@ -512,8 +541,14 @@ func TestLogPoller_BackupPollAndSaveLogsSkippingLogsThatAreTooOld(t *testing.T)
 	logsBatch := 10
 	// Intentionally use very low backupLogPollerDelay to verify if finality is used properly
 	ctx := testutils.Context(t)
-	th := SetupTH(t, true, 0, 3, 2, 1000)
-
+	lpOpts := logpoller.Opts{
+		UseFinalityTag:           true,
+		BackfillBatchSize:        3,
+		RpcBatchSize:             2,
+		KeepFinalizedBlocksDepth: 1000,
+		BackupPollerBlockDelay:   1,
```

Comment:
any reason for changing it from `100` (the previous constant) to `1`?

Reply:
Previously, it was overriding the 100 with 0... but since 0 now means disabled it has to be 1 instead:

```
// Intentionally use very low backupLogPollerDelay to verify if finality is used properly
backupLogPollerDelay := int64(0)
...
th.LogPoller.BackupPollAndSaveLogs(ctx, 0)
```
The `backupLogPollerDelay` param passed to `BackupPollAndSaveLogs` was removed in favor of just using `lp.backupLogPollerDelay` as a part of this PR